crates/tauri-bundler/src/bundle/windows/util.rs

```diff
@@ -67,11 +69,26 @@ pub fn download_webview2_offline_installer(base_path: &Path, arch: &str) -> crat
   Ok(file_path)
 }
 
+fn create_agent_and_url(url: &str) -> crate::Result<(ureq::Agent, String)> {
+  match std::env::var("TAURI_BUNDLER_TOOLS_DOWNLOAD_GITHUB_CDN") {
```

Comment:
How about `TAURI_BUNDLER_TOOLS_GITHUB_MIRROR`?

Reply:
I think it's fine; the name of the variable doesn't matter to me, as it doesn't affect the feature itself.

Reply:
cool, then we should rename it to `TAURI_BUNDLER_TOOLS_GITHUB_MIRROR`